configure

```diff
@@ -1239,6 +1242,11 @@ Installation directories:
 Optional Packages:
   --with-PACKAGE[=ARG]    use PACKAGE [ARG=yes]
   --without-PACKAGE       do not use PACKAGE (same as --with-PACKAGE=no)
+  --with-python-sys-prefix
```

Comment:
Now that we are using a more recent version of automake. We probably want to pull in this m4 macro to update the help output that we had before: https://github.com/aws/aws-cli/pull/7565/files#diff-a8b976bc5f63366bb7d38a6e3277c5134a59535dd55169f8427622e970176b70R2